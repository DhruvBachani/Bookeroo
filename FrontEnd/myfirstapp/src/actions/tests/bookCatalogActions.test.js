import React from 'react';
import reducer from '../../../src/reducers'
import { shallow } from 'enzyme';
import configureMockStore from 'redux-mock-store';

const mockStore = configureMockStore();
const store = mockStore([]);

const createBook = () => ({
    type: 'GET_ERRORS'
})

describe('bookCatalogActions', () => {
    it('creates GET_ERRORS when a book cannot be created', () => {
        store.dispatch(createBook());
        expect(store.getActions()).toMatchSnapshot();
    })
})