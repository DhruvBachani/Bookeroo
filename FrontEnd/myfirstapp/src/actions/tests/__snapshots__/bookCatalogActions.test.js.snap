// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`bookCatalogActions creates GET_ERRORS when a book cannot be created 1`] = `
Array [
  Object {
    "type": "GET_ERRORS",
  },
]
`;
